log/logfmt_logger.go

```diff
@@ -0,0 +1,29 @@
+package log
+
+import (
+	"io"
+
+	"gopkg.in/logfmt.v0"
+)
+
+type logfmtLogger struct {
+	w io.Writer
+}
+
+// NewLogfmtLogger returns a logger that encodes keyvals to the Writer in
+// logfmt format.
+func NewLogfmtLogger(w io.Writer) Logger {
+	return &logfmtLogger{w}
+}
+
+func (l logfmtLogger) Log(keyvals ...interface{}) error {
+	b, err := logfmt.MarshalKeyvals(keyvals...)
```

Comment:
I'd like to see values encoded directly to the writer, rather than using this intermediate buffer. [From what I can see](https://github.com/go-logfmt/logfmt/blob/c50dc9aa845e681529d8e49d3dc11ec0195aacac/logfmt.go#L31) this should be possible?